docs/production.md

```diff
@@ -412,6 +413,68 @@ The verification server uses the Google Identity Platform for authorization.
 
 3. Visit [Google Identity Platform Settings](https://console.cloud.google.com/customer-identity/settings) and ensure that 'Enable create (sign-up)' and 'Enable delete' are unchecked. This system is intended to be invite-only and these flows are handled by administrators.
 
+## End-to-end test runner
+
+Log in as a system admin and view realms, select the `e2e-test-realm`.
+
+![system admin realms](images/e2e/image01.png)
+
+Join the realm
+
+![join e2e-test-realm](images/e2e/image02.png)
+
+Select the "Back to e2e-test-realm" link
+
+![back to e2e-test-realm](images/e2e/image03.png)
+
+Select `Signing Keys` from the drop down menu
+
+![select signing keys](images/e2e/image04.png)
+
+Create a new (initial) realm specific signing key
+
+![create realm signing keys](images/e2e/image05.png)
+
+Set the issuer and audience values (`iss`/`aud`) according to your domain settings, and click `Save realm certificate settings`. You will need this information
```

Comment:
What does "according to your domain settings" mean in this context?

Reply:
added suggested values